roles/sync/templates/config.yml.j2

```diff
@@ -3,8 +3,8 @@ version: {{ sync.version }}
 
 # ActiveMQ config
 messaging:
-    username: {{ activemq_user | default('admin') }}
-    password: {{ activemq_password | default('admin') }}
+    username: {{ activemq_username }}
```

Comment:
template will break if no external host is set and activemq_* param are present.